spec/spec_helper.rb

```diff
@@ -5,11 +5,30 @@
 RSpec.configure do |config|
   config.before(:each) do
     Chef::Config.reset
-    { :profitbricks_username => 'farid.shah@profitbricks.com',
-      :profitbricks_password => 'spc2015',
-      :profitbricks_url => 'https://spc.profitbricks.com'
+    {
```

Comment:
Let's switch to the other assignment style here as well: `key: 'value'`

Reply:
You commented lines that were removed